user/deployment/pages.md

```diff
@@ -19,7 +19,7 @@ For a minimal configuration, add the following to your `.travis.yml`:
 deploy:
   provider: pages
   skip-cleanup: true
-  github-token: $GITHUB_TOKEN  # Set in travis-ci.org dashboard, marked secure
+  github-token: $GITHUB_TOKEN  # Set in travis-ci.com dashboard, marked secure
```

Comment:
I think instead of the travis-ci.com dashboard, we should say: 

"Set in the settings page of your repository, marked secure" 

What do you think?

Reply:
Perfect